master/internal/api_experiment.go

```diff
@@ -121,60 +120,19 @@ func (a *apiServer) GetExperiments(
 		orderExpr = fmt.Sprintf("id %s", sortByMap[req.OrderBy])
 	}
 
-	// Query the database.
 	resp := &apiv1.GetExperimentsResponse{}
-	type experimentsWrapper struct {
-		Count       int32  `db:"count"`
-		Experiments []byte `db:"experiments"`
-	}
-	res := experimentsWrapper{}
-	if err := a.m.db.QueryF(
+	return resp, a.m.db.QueryProtof(
 		"get_experiments",
 		[]interface{}{orderExpr},
-		&res,
+		resp,
 		stateFilterExpr,
 		archivedExpr,
 		userFilterExpr,
 		labelFilterExpr,
 		req.Description,
 		req.Offset,
 		req.Limit,
```

Comment:
BTW, do you mind setting the limit to be 100 here? That fix for [DET-4917](https://determinedai.atlassian.net/browse/DET-4917) also needs to get into the next release.

Reply:
It's fixed already, but in the `public.page_info` function. It defaults to all like it used to.